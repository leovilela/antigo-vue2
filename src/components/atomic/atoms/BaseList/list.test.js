import { shallowMount } from "@vue/test-utils";
import List from "../BaseList/list";

const factory = () => shallowMount(List, {
  stubs: ["Base-List"],
});

describe("List", () => {
  test("mounts properly", () => {
    const wrapper = factory();
    expect(wrapper.isVueInstance()).toBeTruthy();
  });

  test("renders properly", () => {
    const wrapper = factory();
    expect(wrapper.html()).toMatchSnapshot();
  });
});
