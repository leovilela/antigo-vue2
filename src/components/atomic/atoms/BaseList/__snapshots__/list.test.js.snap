// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List renders properly 1`] = `
<div>
  <ul>
    <li>3211827</li>
    <li>Nome da Loja Ltda</li>
    <li>José Maria da Silva</li>
    <li>52.980,00</li>
    <li>01.08.2019</li>
    <li>15.08.2019</li>
    <li>Finalizado</li>
    <li>Ações</li>
  </ul>
  <ul>
    <li>3211827</li>
    <li>Nome da Loja Parafuzeta Ltda</li>
    <li>José Maria da Silva Nostragildo Pereira</li>
    <li>52.980,00</li>
    <li>01.08.2019</li>
    <li>15.08.2019</li>
    <li>Finalizado</li>
    <li>Ações</li>
  </ul>
</div>
`;
